### ISSUE TITLE
TraceLog.IsKernelAddress makes wrong assumptions for none Windows platforms.

### ISSUE THREAD

**Issue by lateralusX** [microsoft/perfview#1445]
Runing EventPipe on Android Arm32 will produce trace files that can have code address in high address space. Trying to convert that nettrace file to SpeedScope or trying to load it in PerfView will give incomplete stack traces (without any method information). After digging through the parser code I located the issue to be and assumption done in TraceLog.IsKernelAddress where it assumes the following:

```
if (pointerSize == 4)
{
    return ip >= 0x80000000;
}

return ip >= 0xFFFF000000000000;        // TODO I don't know what the true cutoff is.
```

and that could fail on all none Windows platforms. Eliminating this check makes it possible to load and analyse the file as expected.

This is a blocking issue for .net6 since we will roll up a number of new platforms into EventPipe and nettrace (Android/iOS/x64/x86/Arm64/Arm32) that could have code located in high address ranges. The check is also not relevant for nettrace files since they currently only includes managed methods and no native frames.

**Comment by lateralusX** [comment]
//CC @brianrob @josalem

**Comment by lateralusX** [comment]
Fixed by #1480.
